src/main/resources/org/jenkinsci/plugins/registry/notification/token/ApiTokens/config.jelly

```diff
@@ -32,9 +32,9 @@
                                     <input type="hidden" class="dockerhub-api-token-uuid-input" name="apiTokenUuid" value="${apiToken.uuid}" />
                                     <f:textbox clazz="dockerhub-api-token-name-input" name="apiTokenName" placeholder="${%Access token name}"/>
                                     <span class="dockerhub-new-api-token-value hidden"><!-- to be filled by JS --></span>
-                                    <span class="yui-button dockerhub-api-token-save-button">
+                                    <span class="dockerhub-api-token-save-button">
```

Comment:
why is the `span` needed?

Reply:
https://github.com/search?q=repo%3Ajenkinsci%2Fdockerhub-notification-plugin%20dockerhub-api-token&type=code maybe because of that?

Reply:
> https://github.com/search?q=repo%3Ajenkinsci%2Fdockerhub-notification-plugin%20dockerhub-api-token&type=code maybe because of that?

Yes. Without docker-hub related css, functionality breaks.